hassio/src/hassio-main.ts

```diff
@@ -169,12 +175,14 @@ class HassioMain extends ProvideHassLitMixin(HassRouterPage) {
       return;
     }
 
-    const [supervisorInfo, hostInfo, hassInfo] = await Promise.all([
+    const [supervisorInfo, hostInfo, hassInfo, hassioInfo] = await Promise.all([
```

Comment:
Now we're delaying loading all hassio information until it is fetched. I guess it's fine for this one.

I wonder if in a future PR we should change to a new format where we have a `hassio` object with `{ supervisor, hassio, host, hass }`